weaviate/collection/classes/grpc.py

```diff
@@ -50,6 +51,55 @@ class NearObjectOptions:
     autocut: Optional[int] = None
 
 
+@dataclass
+class Move:
+    force: float
+    objects: Optional[Union[List[UUID], UUID]] = None
+    concepts: Optional[Union[List[str], str]] = None
+
+    def __post_init__(self):
+        if (
+            self.objects is None or (isinstance(self.objects, list) and len(self.objects) == 0)
+        ) and (
+            self.concepts is None
+            or (isinstance(self.concepts, list) and len(self.concepts) == 0) == 0
+        ):
+            raise ValueError("Either objects or concepts need to be given")
+
+        # accept single values, but make them a list
+        if self.objects is None:
+            self.__objects = None
+        elif not isinstance(self.objects, list):
+            self.__objects = [str(self.objects)]
+        else:
+            self.__objects = [str(obj_uuid) for obj_uuid in self.objects]
+
+        if self.concepts is None:
+            self.__concepts = None
+        elif not isinstance(self.concepts, list):
+            self.__concepts = [self.concepts]
+        else:
+            self.__concepts = self.concepts
+
+    @property
+    def objects_list(self) -> Optional[List[str]]:
+        return self.__objects
+
+    @property
+    def concepts_list(self) -> Optional[List[str]]:
+        return self.__concepts
```

Comment:
In light of the `Filters` changes away from `@dataclass`, can this change into the same style too? (Where `__post_init__` is an `__init__` and the class vars become `__init__` func args)